index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700&family=Open+Sans&family=Poppins:ital,wght@1,200&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Header part start-->
+    <header>
+      <nav class="navbar">
+        <img src="images/Logo Placeholder.png" alt="logo" />
+        <a class="toggler" target="_self" href="nav-mobile.html">
+          <img src="images/Icon - Menu.png" alt="icon image" />
+        </a>
+      </nav>
+    </header>
```

Comment:
You are using the image logo on the header, which is not wrong. But it is creating a lot of space. The space between the logo and the header bottom is `12px` but in the current styling, it is more.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219126673-a11c5cca-4773-4848-8d72-61174be9fd6d.png)  | ![image](https://user-images.githubusercontent.com/61361037/219126599-87dfb55c-6fd3-40f0-9231-b03d23723325.png)  |